homeassistant/components/mysensors/cover.py

```diff
@@ -14,6 +15,15 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class CoverState(Enum):
+    """An enumeration of the standard cover states."""
+
+    OPEN = 0
+    OPENING = 1
+    CLOSED = 2
+    CLOSING = 3
```

Comment:
We could also just make these strings?
```suggestion
    OPEN = "open"
    OPENING = "opening"
    CLOSING = "closing"
    CLOSED = "closed"
```

Reply:
Do you have a specific reason for it? I personally prefer to keep enumerations numeric as you can access the enumeration name via Enum.name (at least in terms of the capitalized formal name). I have no issue making the change if you have a use case you can see that would use it.

That said, we probably should have @unique applied to the enum either way.

Reply:
I guess I'm wondering what's the use case for the numbers. There's nothing saying they should be numbers. It's a local definition that we use just here and we don't use the numbers.

Reply:
Numeric ordinals are just the default value. The Enum has to be bound to some constant value, and you are correct, you can use Strings for this local use, it doesn't exactly matter. I tend to prefer sequential integers as they can be used for something like an index to a list of other constants, but as you said, I'm not doing anything with them in this local case.

Reply:
I'm ok with keeping them as numbers.